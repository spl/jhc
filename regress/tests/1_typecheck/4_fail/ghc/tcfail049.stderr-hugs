ERROR "tcfail049.hs":3 - Undefined variable "g"
