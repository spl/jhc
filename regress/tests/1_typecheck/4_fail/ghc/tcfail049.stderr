
tcfail049.hs:3:7: Not in scope: `g'
